### ISSUE TITLE
Proposal: Expand the SRC-44 description field structure by adding a new field referencing country code

### ISSUE THREAD

**Issue by ipr0310** [signum-network/SIPs#77]
I propose that we expand the SRC-44 descriptor to include a new `OPTIONAL` property that refers to a "country code" in the description field structure.

The key would be labeled as `ctry` and the value would be just an `ALPHA-2 Country code`
https://www.iban.com/country-codes

These would be the properties

| **Field Name** | **Required** | **Full Name** | **Value Format** | **Example** | **Rules**                    | **Description**                                          |
|----------------|--------------|---------------|------------------|-------------|------------------------------|----------------------------------------------------------|
| ctry           | NO           | Country Code       | string           | US          | A valid ALPHA-2 country code | A string property which can help a profile expand its digital identity |

**Comment by frankTheTank72** [comment]
Good idea - i just would go for 'cc' as country code, to be conform with the other fields. 

Maybe we can describe the field in that way, that you can enter all three ways from your provided link - as 2 letters , 3 letters or numeric ?!

**Comment by ipr0310** [comment]
> Maybe we can describe the field in that way, that you can enter all three ways from your provided link - as 2 letters , 3 letters or numeric ?!

Well, it may also be possible, but to keep consistency between all the descriptors that can be used by different use cases for specific reasons, it is best to keep one format.

**Comment by frankTheTank72** [comment]
> > Maybe we can describe the field in that way, that you can enter all three ways from your provided link - as 2 letters , 3 letters or numeric ?!
> 
> Well, it may also be possible, but to keep consistency between all the descriptors that can be used by different use cases for specific reasons, it is best to keep one format.

excactly that why we should keep it open for all 3 options - so we not limited upcoming use-cases

**Comment by ohager** [comment]
That country codes refers to what then? I think would only apply to account profile descriptors. Technically, it's possible to use the custom fields: `x-cc` for example
